src/services/test/account-state-engine.test.ts

```diff
@@ -145,7 +145,7 @@ const idResetSuccessfulUpdateSuspended = {
     resetPassword: true,
     reproveIdentity: false,
   },
-  nextAllowableInterventions: ['01', '02', '03', '05', '06', '90']
+  nextAllowableInterventions: ['01', '02', '03', '05', '06', '90', '95']
```

Comment:
could we also add the event itself? For example in the block: `from suspended psw reset required` and anywhere else this transition is applicable